internal/rbd/rbd_attach.go

```diff
@@ -417,6 +424,13 @@ func detachRBDImageOrDeviceSpec(
 
 		return fmt.Errorf("rbd: unmap for spec (%s) failed (%w): (%s)", imageOrDeviceSpec, err, stderr)
 	}
+	if isNbd && logDir != "" {
+		logFile := fmt.Sprintf("%s/rbd-nbd-%s.log", logDir, volumeID)
```

Comment:
Can we have a common helper function to return the log file path?

Reply:
Sure :+1:

Reply:
Done now. Ready for review again :-)